File: 240123/문자열의 특정 위치 찾기 2/find-specific-location-in-spring-2.py
# arr 문자열 5개 초기화
arr = ['apple', 'banana', 'grape', 'blueberry', 'orange']

# input()값 선언
alpha = input()

# 문자열 개수 변수 선언
cnt = 0

# list의 요소를 for문으로 돌려 요소 3 or 4번째 문자와 alpha 요소 같은지 확인, 들어있다면 해당 요소 print 후 카운트
for i in arr:
    if alpha == i[2] or alpha == i[3]:
        print(i)
        cnt += 1

print(cnt)

File: 250429/문자열의 특정 위치 찾기 2/find-specific-location-in-spring-2.js
const fs = require("fs");

const inputSpell = fs.readFileSync(0).toString().trim();

const wordList = ["apple", "banana", "grape", "blueberry", "orange"];

function findThirdAndFourthSpellMatchWordList(spell,list) {
    let resultArray = [];
    for (let i = 0; i < 5; i ++) {
        if (list[i][2] === spell || list[i][3] === spell) {
            resultArray.push(list[i]);
        };
    };
    resultArray.push(resultArray.length);

    return resultArray;
};

console.log(findThirdAndFourthSpellMatchWordList(inputSpell,wordList).join("\n"));